packages/scalar.aspnetcore/Scalar.AspNetCore.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <Version>1.0.1</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="9.0.0-preview.4.24260.3" />

```

Comment:
You shouldn't need a package reference to the `Microsoft.AspNetCore.OpenApi` package in `Scalar.AspNetCore` since you don't rely on any of the APIs in it. Instead, we can assume that the use has installed the package in their web project and exposed the OpenAPI endpoint the Scalar API will point to from there.